cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -27,6 +26,7 @@
     is_ontological_descendant_of,
     read_h5ad,
 )
+from .validation_internals.check_duplicates import check_duplicate_obs
```

Comment:
minor copy change to just have this be consistent with the other subdirectory imports